addon/components/model-form.hbs

```diff
@@ -1,37 +1,35 @@
-{{#if @model}}
```

Comment:
Does it no longer fail to generate a changeset if the model is unavailable on initialization?

Reply:
Not that i noticed no. Did this happen on all forms or just specific ones?

Reply:
It used to happen at least with the project form

Reply:
Project form works as expected :ok_hand:

Reply:
I wonder what wasn't working before. Great if it works now :ok_hand: